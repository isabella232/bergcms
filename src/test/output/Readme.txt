Test cases will write result files to this directory.
