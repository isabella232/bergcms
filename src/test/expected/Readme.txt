This directory contains the files with the expected content of
C++ test cases.
